go.mod

```diff
@@ -25,6 +25,7 @@ require (
 	github.com/mutecomm/go-sqlcipher v0.0.0-20190227152316-55dbde17881f
 	github.com/okzk/sdnotify v0.0.0-20180710141335-d9becc38acbd
 	github.com/pborman/uuid v1.2.0
+	github.com/prometheus/client_golang v1.0.0
```

Comment:
One more thing is that we add another significant dependency.

Reply:
How is that bad?